aplib.net-demo/Assets/Scripts/Enemy/BasicEnemy.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+
+/// <summary>
+/// Very basic enemy class.
```

Comment:
Describe classes like describing the object.
```suggestion
/// Describes a basic enemy with health.
```